### PR TITLE
Changed footpint name to be more generic

### PR DESCRIPTION
**Pull request by jneiva08**
Small changed on the footprint name to be generic.

### CODE REVIEW

**Comment by myfreescalewebpage** [comment]
OK to be merged
Joel

**Comment by jneiva08** [comment]
Have added parameters to generate the same footprint but with suffix "_H1.1mm" earlier commit as changing the name to suffix "_H1.2mm"

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/lga.yaml [hunk 1]
```diff
@@ -175,7 +175,40 @@ NXP-8_3x5mm_P1.25mm_LayoutBorder2x4y:
...
+  suffix: '_H1.2mm'
+  #include_suffix_in_3dpath: 'False'
+
+NXP-8_3x5mm_P1.25mm_LayoutBorder2x4y_H1.1mm:
+  device_type: 'LGA'
+  library: Package_LGA
+  use_name_format: 'LGA'
+  manufacturer: 'NXP'
+  #part_number: 'MPL115A1'
+  size_source: 'https://www.nxp.com/docs/en/data-sheet/MPL3115A2.pdf'
```
Comment (poeschlr) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/lga.yaml]: please add #page=42 to point directly to the correct page.

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/lga.yaml [hunk 2]
```diff
@@ -175,7 +175,40 @@ NXP-8_3x5mm_P1.25mm_LayoutBorder2x4y:
...
+  body_size_x:
+    nominal: 3
+    tolerance: 0
+  body_size_y:
+    nominal: 5
+    tolerance: 0
+  body_height:
+    maximum: 1.2
```
Comment (poeschlr) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/lga.yaml]: given as 1.1 with 0.1 tolerance so please use the same here

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/lga.yaml [hunk 3]
```diff
@@ -175,7 +175,40 @@ NXP-8_3x5mm_P1.25mm_LayoutBorder2x4y:
...
+  #part_number: 'MPL3115A2'
+  size_source: 'https://www.nxp.com/docs/en/data-sheet/MPL3115A2.pdf#page=42'
+  body_size_x:
+    nominal: 3
+    tolerance: 0
+  body_size_y:
+    nominal: 5
+    tolerance: 0
+  body_height:
+    maximum: 1.1
```
Comment (poeschlr) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/lga.yaml]: use nominal: 1.1 and tolerance: 0.1

**Comment by poeschlr** [comment]
@pointhi this can be merged as soon as the height parameter is fixed. (the footprint has already been merged as it is not influenced by the parameter prepared for 3d modeling reuse.)

**Comment by jneiva08** [comment]
hope get right this time

**Comment by jneiva08** [comment]
Thanks

**Pull request merged**